Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/storage/repo/DefaultContactDiaryRepository.kt

```diff
@@ -126,8 +126,10 @@ class DefaultContactDiaryRepository @Inject constructor(
     ) {
         executeWhenIdNotDefault(visitId) {
             val original = contactDiaryLocationVisitDao.entityForId(visitId)
-            val updatedVisit = update(original.toContactDiaryLocationVisit())
-            contactDiaryLocationVisitDao.update(updatedVisit.toContactDiaryLocationVisitEntity())
+            original?.let {
```

Comment:
Maybe add a comment in which case `original` is actually `null` would be helpful.

Reply:
updated